examples/topics/geo/tile_xyzservices.py

```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations 
+and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh
+with a map background using tile_providers.The chart shows a map of a specific geographical region, 
+which is defined by the range bounds supplied in web mercator coordinates.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure, bokeh.tile_providers.get_provider, bokeh.tile_provider.Vendors
+    :refs: :ref:`ug_topics_geo_tile_provider_maps`
+    :keywords: Bokeh, tile services, XYZ, mapping, interactive, geography
```

Comment:
"bokeh" and "interactive" are implicit in all the examples
```suggestion
    :keywords: map plot, tiles, geo
```

Reply:
@bryevdv all changes has been made